src/enum.ts

```diff
@@ -0,0 +1,5 @@
+export enum DashboardDisplay{
+  Listings ='Listings',
+  Tenants ='Tenants',
+  Chat ='Chat'
+}
```

Comment:
Consider running `npm run format` to auto-format your code :stuck_out_tongue:

Reply:
Oops! I'll do that now :)